src/containers/projects-table/projects-table.tsx

```diff
@@ -88,7 +89,9 @@ const ProjectsTable: React.FC = () => {
   // define what happens during various row events
   const rowEvents: RowEventHandlerProps = {
     onClick: (e, row, rowIndex) => {
-      history.push(`/projects/${row.wbsNum}`);
+      history.push({
+        pathname: `/projects/${row.wbsNum}`
+      });
```

Comment:
Since we removed the state field, let's put it back the way it was before so as to not introduce unnecessary changes